recipe/meta.yaml

```diff
@@ -28,17 +28,21 @@ requirements:
     - argh >=0.24.1
     - pathtools >=0.1.1
     - pyyaml >=3.10
+    - pyobjc-framework-cocoa >=4.2.2  # [osx]
+    - pyobjc-framework-fsevents >=4.2.2  # [osx]
 
 test:
+  requires:
+    - pip
   imports:
     - watchdog
     - watchdog.observers
+    - watchdog.observers.fsevents  # [osx]
     - watchdog.tricks
     - watchdog.utils
   commands:
+    - python -m pip check
```

Comment:
What does this test do?

Reply:
Checks for the consistency of the Python requirements as declared by the package. This will fail if a dependency was forgotten, in the case of `watchdog`, it would have errored on the missing `pyobjc-framework-cocoa` on OSX.